templates/home.mlx

```diff
@@ -187,12 +187,22 @@ let venue_section =
 
   </div>
 
-let sponsors_section =
+let sponsors_section = 
   <div id="sponsors" class_="container mx-auto px-6 py-20">
-    (h2 "Sponsors")
-
-    <div class_="mt-6">
-      (string "TBA")
+    (h2 ~classes:"text-center" "Sponsors & Partners")
+
+    <div style="gap:2rem" class_="flex justify-center mt-4"> 
```

Comment:
for some reason half of the tailwind css classes I want to use are simply not supported. What version of tailwind it is using? No support for grid-cols, gap,etc